DESCRIPTION

```diff
@@ -30,6 +30,7 @@ Imports:
   broom,
   corrr,
   Cubist,
+  DALEXtra,
```

Comment:
Can you add `forcats` and `stringr`?

Reply:
They get installed with tidyverse; is there a reason to have them separately here in `DESCRIPTION`?

Reply:
Yes, for visibility and in case we need specific versions. We also use this list for the package version table in the preface.